wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,75 @@
+osu! uses automated metrics combined with manual confirmation.
 
-The staff never places an incorrect ban for cheating. Before they do any actions, they make sure it first 100%.
+The team never places an incorrect ban for cheating. Before they take any action, they make sure it is 100% certain that you cheated.
 
-My friend has been banned for cheating! They definitely wouldn't cheat! You are wrong!
---------------------------------------------------------------------------------------
 
-It may be hard to come to terms with, but you will have to trust the staff. peppy is not trying to tear apart friendships but he knows that nearly no one can be trusted. Even those people who are close to you that you may never suspect can be just acting behind you or any other person.
+## My friend has been banned for cheating! They definitely wouldn't cheat! You are wrong!
+We understand that you trust your friends, but you will have to believe the team. Punishments aren't issued unless there is sufficient evidence to support the decision. Whether or not they have cheated, it shouldn't affect your friendship!
 
-Prove that I/they were cheating! Tell us how you know!
-------------------------------------------------------
 
-The staff cannot say on how can they prove. Giving away the countermeasures would make them useless in further detecting cheaters.
+## Prove that I/they were cheating! Tell us how you know!
+The team cannot say on how can they prove. Giving away the countermeasures would make them useless in further detecting cheaters.
 
-I'll make a video to prove my innocence!
-----------------------------------------
 
+## I'll make a video to prove my innocence!
 Sorry but if you have been banned, they are already 100% sure you are cheating. A video will not help.
 
-I found someone who is definitely cheating. I can tell by their replay/score! What should I do?
------------------------------------------------------------------------------------------------
 
-Stay calm.
+## I found someone who is definitely cheating. I can tell by their replay/score! What should I do?
+Stay calm and **don't discuss their play in public chat**. Don't give them undue attention or ask others to spectate them.
+You should report the user and not mention them anywhere. There are three ways to do so.
+
+### User Profile
+On the user's profile, a `Report User` button will appear when you move your cursor to their avatar. Write any information that can prove that the user is doing misbehaviour. This is the preferred way for reporting foul play.
 
-**DON'T discuss their play in public chat**.
+*Take note of the selected game mode before reporting*
 
-Don't give them undue attention or ask others to spectate them.
+### Beatmap Page
+In beatmap pages, you can report a user in the top leaderboards by clicking the `Report` button on the far right of their score.
 
-Click the "Report" link at the right side of their score on the beatmap info (website)
+### In-game chat
```

Comment:
don't encourage use of in-game report for cheating stuff since profile provides much more information to us overall